packages/replay-internal/src/util/createPerformanceEntries.ts

```diff
@@ -60,7 +60,7 @@ interface LayoutShiftAttribution {
 
 interface Attribution {
```

Comment:
Can we use `WebVitalData['attributions']` instead of a separate type that needs to be kept in sync?